server/service/src/sync/sync_on_central/mod.rs

```diff
@@ -75,10 +90,13 @@ pub async fn pull(
     );
     log::debug!("Sending records as central server: {:#?}", records);
 
+    let is_last_batch = total_records <= batch_size as u64;
+
     Ok(SyncBatchV6 {
         total_records,
         end_cursor,
         records,
+        is_last_batch,
```

Comment:
Kind of icky having to pass is_last_batch for pull, seeing as we don't use it on the other side... oh well?